kaspi_app/SwiftUI/ButtonGridSwiftUI.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ButtonGridSwiftUI.swift
+//  kaspi_app
+//
+//  Created by Yersin Kazybekov on 20.12.2023.
+//
+
+import Foundation
+import SwiftUI
+
+let layout = [
+        GridItem(.fixed(74)),
+        GridItem(.fixed(74))
+]
+
+let buttonCollection = [
```

Comment:
for now keep it here, but in future we will move it to the Model/Presenter, because it is data, not UI